CHANGELOG.md

```diff
@@ -1,6 +1,132 @@
 # Changelog
 Notable changes to the framework should be documented here
 
+3.12.0b1:
+	- Avoid executing self upgrade policy unnecessarily (ENT-3592)
+	- Add amazon_linux class to yum package module
+	- Introduce ability to set policy update bundle via augments (CFE-2687)
+	- Localize delete tidy in ha update policy (ENT-3659)
+	- Improve context notifying user of missing policy update bundle
+	  (ENT-3624)
+	- Configure ignore_missing_inputs and ignore_missing_bundles via augments
+	  (CFE-2773)
+	- Change class identifying runagent initiated executions from cfruncommand to cf_runagent_initated
+	- Support enablerepo and disablerepo options in yum package_module
+	  (CFE-2806)
+	- Fix cf-runagent during 3.7.x -> 3.10.x migration
+	  (CFE-2776, CFE-2781, CFE-2782)
+	- Makes it possible to tune policy master_location via augments in update policy
+	  (ENT-3692)
+	- Fix inventory for total memory on AIX (CFE-2797)
+	- Do not manage redis since it's no longer used (ENT-2797)
+
+3.12.0a1:
```

Comment:
We didnt release alpha1, so I think this should just be listed in the b01 section.